lib/puppetlabs_spec_helper/rake_tasks.rb

```diff
@@ -292,11 +292,17 @@ def max_thread_limit
     end
     next if File::exists?(target)
 
+    # https://tickets.puppetlabs.com/browse/PUP-4884
+    working_dir = File.expand_path('spec/fixtures/work-dir')
+    working_dir = working_dir.gsub(/\\/, '/')
+    target_dir = File.expand_path('spec/fixtures/modules')
+    target_dir = target_dir.gsub(/\\/, '/')
+
     command = "puppet module install" + ref + flags + \
       " --ignore-dependencies" \
       " --force" \
-      " --module_working_dir spec/fixtures/module-working-dir" \
-      " --target-dir spec/fixtures/modules #{remote}"
+      " --module_working_dir #{working_dir}" \
+      " --target-dir #{target_dir} #{remote}"
```

Comment:
ah! this was the piece I missed. Apologies! :man_facepalming: 

If you could put the "The problem with the relative path..." sentence into a comment in the code, that would go a long way towards making it clear for future-us to understand what's going on here.